src/display/api.js

```diff
@@ -3021,6 +3021,9 @@ class WorkerTransport {
   }
 
   getDocJSActions() {
+    if (typeof PDFJSDev !== "undefined" && PDFJSDev.test("GECKOVIEW")) {
+      return this.#cacheSimpleMethod("GetDocJSActions");
+    }
     return this.messageHandler.sendWithPromise("GetDocJSActions", null);
```

Comment:
I'd just do the caching *unconditionally*, since otherwise the GV code-path isn't actually covered by tests.
Furthermore, given that most PDF documents don't have scripting we'd only be caching a `null` value in an already existing `Map` hence I don't think it matters in the grand scheme of things.
```suggestion
    return this.#cacheSimpleMethod("GetDocJSActions");
```